lib/kube/proxy/auth.go

```diff
@@ -174,6 +174,19 @@ func extractKubeCreds(ctx context.Context, cluster string, clientCfg *rest.Confi
 	if err != nil {
 		return nil, trace.Wrap(err, "failed to generate TLS config from kubeconfig: %v", err)
 	}
+	if tlsConfig == nil {
+		cc := rest.AnonymousClientConfig(clientCfg)
+		if len(cc.CAData) != 0 {
+			cc.CAData = []byte("REDACTED")
+		}
+		if len(cc.CertData) != 0 {
+			cc.CertData = []byte("REDACTED")
+		}
+		if len(cc.KeyData) != 0 {
+			cc.KeyData = []byte("REDACTED")
+		}
```

Comment:
@lxea Is this still needed with `AnonymousClientConfig` being used?

Reply:
Only CAData is still needed here with AnonymousClientConfig